draft-iab-protocol-maintenance.md

```diff
@@ -245,7 +245,10 @@ the negative effects on future implementations and the protocol as a whole.
 
 For a protocol to have sustained viability, it is necessary for both
 specifications and implementations to be responsive to changes, in addition to
-handling new and old problems that might arise over time.
+handling new and old problems that might arise over time. Additionally,
+since implementations now already benefit from periodic updates to mitigate
+security vulnerabilities, the cost of reacting to changes is much lower than it
+was in the past.
```

Comment:
I'm less sure about this.  It's a common argument I like to use, but I would want to inject a little more nuance into this.

When I've talked to people on the routing side of the IETF, they are adamant that the process for compatibility changes is very different to the one they use for security fixes.  In other words, it's not as simple as you make out (or at least people don't believe that it is that simple...).  Adding this text would add ammunition to their complaints about this draft.

Reply:
That's totally fair. How about we don't mention security and just say that software update mechanisms have gotten better over the years making protocol reaction changes easier?

Reply:
That might be a better angle on this, yes.

Reply:
Done